syntax = "proto3";
package opb;

option go_package = ".;opb";

message SubscribeScanLoginRequest{
  string ToUserName = 1;  // target
  string FromUserName = 2;  // source
  string MsgType = 3;  // type
  string Event = 4;  // event
  string EventKey = 5;  // key
  string Ticket = 6;  // ticket
}

message SubscribeScanLoginResponse{
  // 是否成功
  bool OK = 1;
  // 消息内容
  string Msg = 2;
}

service WXAdmin {
  rpc SubscribeScanLogin(SubscribeScanLoginRequest) returns (SubscribeScanLoginResponse) {}
}